titus_isolate/model/workload.py

```diff
@@ -18,7 +18,8 @@ def __init__(
             command,
             entrypoint,
             job_type,
-            workload_type):
+            workload_type,
+            opportunistic):
```

Comment:
I believe @fabiokung is going to be exposing not just whether a workload is opportunistic or not but how many opportunistic threads it intends to consume.

So I would make construction take the opportunistic thread count, then put methods on the Workload object like:
```python
def is_opportunistic() -> bool:
    pass

def opportunistic_thread_count() -> int:
    pass
```

Reply:
done. will need to sync with fabio about the label names.